pkg/util/util.go

```diff
@@ -125,3 +125,14 @@ func GetEsHostname(opts map[string]string) string {
 	urlArr := strings.Split(urls, ",")
 	return urlArr[0]
 }
+
+// HasArg returns whether the slice has an entry prefixed by the given name
+func HasArg(name string, args []string) bool {
```

Comment:
There is a similar function in  `elasticsearch.go`. Actually it was exactly the same to the moment I had to check the exact argument.  I would suggest returning the item and removing it from es.go. 

 https://github.com/jaegertracing/jaeger-operator/blob/c1e8337d5b65861b585b5fe2abf5252c34a4853b/pkg/storage/elasticsearch.go#L208

```
func findItem(prefix string, arr []string) string {
	for _, a := range arr {
		if strings.HasPrefix(a, prefix) {
			return a
		}
	}
	return ""
}
```